examples/chat/src/listener.js

```diff
@@ -33,7 +34,7 @@ async function run() {
 
   // Output listen addresses to the console
   console.log('Listener ready, listening on:')
-  peerListener.multiaddrs.forEach((ma) => {
+  nodeListener.addresses.listen.forEach((ma) => {
```

Comment:
Should use the new multiaddrs getter
```suggestion
  nodeListener.multiaddrs.forEach((ma) => {
```